packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -238,6 +266,7 @@ export const genCrossVersionCompatConfig = (): CompatConfig[] => {
 						compatVersion: resolvedCreateVersion,
 						createWith: createVersion,
 						loadWith: loadVersion,
+						loadVersion: resolvedLoadVersion,
```

Comment:
We need both versions (load and create) to compare the minimum of those against the specified minVersion.

Reply:
You can avoid adding `loadVersion` by generating the `resolvedLoadVersion` from `loadWith` in `isCompatVersionBelowMinVersion` as it's only used there.